plugins/servicenow/help.md

```diff
@@ -307,15 +367,22 @@ This action is used to associate a file with a ServiceNow Incident.
 
 |Name|Type|Default|Required|Description|Enum|Example|
 |----|----|-------|--------|-----------|----|-------|
-|attachment_name|string|None|True|Name of the attachment in the ServiceNow instance|None|None|
-|base64_content|bytes|None|True|Content of the attachment, encoded into Base64|None|None|
-|mime_type|string|None|True|MIME type (a.k.a. content type) of the file to be attached|['text/plain (.txt)', 'text/html (.html)', 'application/rtf (.rtf)', 'application/pdf (.pdf)', 'application/msword (.doc)', 'application/vnd.ms-powerpoint (.ppt)', 'image/bmp (.bmp)', 'image/gif (.gif)', 'image/jpeg (.jpg)', 'image/png (.png)', 'image/tiff (.tiff)', 'OTHER']|None|
-|other_mime_type|string|None|False|User-specified MIME type not in the enumerated list|None|None|
-|system_id|string|None|True|System ID of the Incident record to which the file will be attached|None|None|
+|attachment_name|string|None|True|Name of the attachment in the ServiceNow instance|None|Example name|
+|base64_content|bytes|None|True|Content of the attachment, encoded into Base64|None|UmFwaWQ3IEluc2lnaHRDb25uZWN0Cg==|
+|mime_type|string|None|True|MIME type (a.k.a. content type) of the file to be attached|['text/plain (.txt)', 'text/html (.html)', 'application/rtf (.rtf)', 'application/pdf (.pdf)', 'application/msword (.doc)', 'application/vnd.ms-powerpoint (.ppt)', 'image/bmp (.bmp)', 'image/gif (.gif)', 'image/jpeg (.jpg)', 'image/png (.png)', 'image/tiff (.tiff)', 'OTHER']|text/plain (.txt)|
```

Comment:
It's nice to have the types written out afterward, but people may get confused and try to enter the whole string, for example instead of text/plain they may enter "text/plain (.txt)" which is not a MIME type

Reply:
I think the first section is good, but in my opinion the (.txt) should be removed from the actual example

Reply:
@pmara-r7 In this plugin user have to use MIME type in form like in example. Here is this field from plugin.spec.yml:
```
      mime_type:
        title: MIME Type
        description: MIME type (a.k.a. content type) of the file to be attached
        type: string
        enum:
        - text/plain (.txt)
        - text/html (.html)
        - application/rtf (.rtf)
        - application/pdf (.pdf)
        - application/msword (.doc)
        - application/vnd.ms-powerpoint (.ppt)
        - image/bmp (.bmp)
        - image/gif (.gif)
        - image/jpeg (.jpg)
        - image/png (.png)
        - image/tiff (.tiff)
        - OTHER
```
So if we use this without "(.txt)" there is an ValidationError. User can also use input `other_mime_type` where user can type "text/plain" - without "(.txt)".

Reply:
I think it's okay to include the file extensions here. It should be safe to assume that our users understand the difference between MIME types and file extensions.